wasm/src/programs/manager/deploy.rs

```diff
@@ -207,4 +213,138 @@ impl ProgramManager {
+    #[allow(clippy::too_many_arguments)]
+    pub async fn devnode_deploy(
+        private_key: &PrivateKey,
+        program: &str,
+        priority_fee_credits: f64,
+        fee_record: Option<RecordPlaintext>,
+        url: Option<String>,
+        imports: Option<Object>,
+        offline_query: Option<OfflineQuery>,
+    ) -> Result<Transaction, String> {
+        log("Creating deployment transaction");
+        let mut process_native = ProcessNative::load_web().map_err(|err| err.to_string())?;
+        let process = &mut process_native;
+
+        log("Checking program has a valid name");
+        let program = ProgramNative::from_str(program).map_err(|err| err.to_string())?;
+        let program_id = program.id();
+
+        log("Checking program imports are valid and add them to the process");
+        ProgramManager::resolve_imports(process, &program, imports)?;
+        let rng = &mut StdRng::from_entropy();
+
+        log("Creating deployment");
+        let node_url = url.as_deref().unwrap_or(DEFAULT_URL);
+
+        // Create deployment without synthesizing keys and generating certificates.
+        assert!(!program.functions().is_empty(), "Program must have at least one function");
+        let mut verifying_keys = Vec::with_capacity(program.functions().len());
+        for function_name in program.functions().keys() {
+            let (verifying_key, certificate) = {
+                // Sample a dummy verifying key for each function.
+                let verifying_key = VerifyingKeyNative::from_str(
+                    "verifier1qygqqqqqqqqqqqyvxgqqqqqqqqq87vsqqqqqqqqqhe7sqqqqqqqqqma4qqqqqqqqqq65yqqqqqqqqqqvqqqqqqqqqqqgtlaj49fmrk2d8slmselaj9tpucgxv6awu6yu4pfcn5xa0yy0tpxpc8wemasjvvxr9248vt3509vpk3u60ejyfd9xtvjmudpp7ljq2csk4yqz70ug3x8xp3xn3ul0yrrw0mvd2g8ju7rts50u3smue03gp99j88f0ky8h6fjlpvh58rmxv53mldmgrxa3fq6spsh8gt5whvsyu2rk4a2wmeyrgvvdf29pwp02srktxnvht3k6ff094usjtllggva2ym75xc4lzuqu9xx8ylfkm3qc7lf7ktk9uu9du5raukh828dzgq26hrarq5ajjl7pz7zk924kekjrp92r6jh9dpp05mxtuffwlmvew84dvnqrkre7lw29mkdzgdxwe7q8z0vnkv2vwwdraekw2va3plu7rkxhtnkuxvce0qkgxcxn5mtg9q2c3vxdf2r7jjse2g68dgvyh85q4mzfnvn07lletrpty3vypus00gfu9m47rzay4mh5w9f03z9zgzgzhkv0mupdqsk8naljqm9tc2qqzhf6yp3mnv2ey89xk7sw9pslzzlkndfd2upzmew4e4vnrkr556kexs9qrykkuhsr260mnrgh7uv0sp2meky0keeukaxgjdsnmy77kl48g3swcvqdjm50ejzr7x04vy7hn7anhd0xeetclxunnl7pd6e52qxdlr3nmutz4zr8f2xqa57a2zkl59a28w842cj4783zpy9hxw03k6vz4a3uu7sm072uqknpxjk8fyq4vxtqd08kd93c2mt40lj9ag35nm4rwcfjayejk57m9qqu83qnkrj3sz90pw808srmf705n2yu6gvqazpvu2mwm8x6mgtlsntxfhr0qas43rqxnccft36z4ygty86390t7vrt08derz8368z8ekn3yywxgp4uq24gm6e58tpp0lcvtpsm3nkwpnmzztx4qvkaf6vk38wg787h8mfpqqqqqqqqqqt49m8x",
+                    ).map_err(|err| err.to_string())?;
+
+                // Sample a dummy certificate.
+                let certificate = CertificateNative::from_str(
+                    "certificate1qyqsqqqqqqqqqqxvwszp09v860w62s2l4g6eqf0kzppyax5we36957ywqm2dplzwvvlqg0kwlnmhzfatnax7uaqt7yqqqw0sc4u"
```

Comment:
Same here, let's check if it's okay to have a single cert, if so this should be a constant and not hardcoded.